#!/usr/bin/python3
def raise_exception():
    input = 9 
    input2 = "a" 
    result = input - input2 
    return result